lms/djangoapps/bulk_email/tasks.py

```diff
@@ -459,6 +480,7 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
             # That way, the to_list will always contain the recipients remaining to be emailed.
             # This is convenient for retries, which will need to send to those who haven't
             # yet been emailed, but not send to those who have already been sent to.
+            recipient_no += 1
```

Comment:
nit:  I first read 'recipient_no' as "no recipient".  Might be clearer to use recipient_num.